automation/network-security-group/template.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "networkSecurityGroupName": {
+            "type": "string",
+            "metadata": {
+                "description": "This is the name of the network security group"
+            }
+        },
+        "location": {
```

Comment:
adding allowedValues options here would limit user input making it more safe

Reply:
i added this and tested it